src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,94 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+variables [topological_space α] [normed_lattice_add_comm_group β]
+
+-- Can we infer this from https://github.com/leanprover-community/mathlib/blob/f29b0b49badab1bcbe338cb79d102e36e79dce09/src/topology/continuous_function/basic.lean#L114 ?
```

Comment:
You can if you prove that `to_continuous_map` is injective and use it in `partial_order.lift`.
```suggestion
```

Reply:
Thanks - would that be better, or is it okay as it is?

Reply:
It's okay as it is. I don't see a practical advantage to having the order come from the continuous functions.